beets/ui/commands.py

```diff
@@ -255,32 +262,180 @@ def penalty_string(distance, limit=None):
+            url = ui.colorize("text_faint", f"{self.match.info.data_url}")
+            print_(self.indent_header + url)
+
+    def show_match_details(self):
+        """Print out the details of the match, including changes in album name
+        and artist name.
+        """
+        # Artist.
+        artist_l, artist_r = self.cur_artist or "", self.match.info.artist
+        if artist_r == VARIOUS_ARTISTS:
+            # Hide artists for VA releases.
+            artist_l, artist_r = "", ""
+        if artist_l != artist_r:
+            artist_l, artist_r = ui.colordiff(artist_l, artist_r)
+            # Prefix with U+2260: Not Equal To
+            left = {
+                "prefix": ui.colorize("changed", "\u2260") + " Artist: ",
+                "contents": artist_l,
+                "suffix": "",
+            }
+            right = {"prefix": "", "contents": artist_r, "suffix": ""}
+            self.print_layout(self.indent_detail, left, right)
+
+        else:
+            print_(self.indent_detail + "*", "Artist:", artist_r)
+
+        if self.cur_album:
+            # Album
+            album_l, album_r = self.cur_album or "", self.match.info.album
+            if (
+                self.cur_album != self.match.info.album
+                and self.match.info.album != VARIOUS_ARTISTS
+            ):
+                album_l, album_r = ui.colordiff(album_l, album_r)
+                # Prefix with U+2260: Not Equal To
+                left = {
+                    "prefix": ui.colorize("changed", "\u2260") + " Album: ",
+                    "contents": album_l,
+                    "suffix": "",
+                }
+                right = {"prefix": "", "contents": album_r, "suffix": ""}
+                self.print_layout(self.indent_detail, left, right)
+            else:
+                print_(self.indent_detail + "*", "Album:", album_r)
+        elif self.cur_title:
+            # Title - for singletons
+            title_l, title_r = self.cur_title or "", self.match.info.title
+            if self.cur_title != self.match.info.title:
+                title_l, title_r = ui.colordiff(title_l, title_r)
+                # Prefix with U+2260: Not Equal To
+                left = {
+                    "prefix": ui.colorize("changed", "\u2260") + " Title: ",
+                    "contents": title_l,
+                    "suffix": "",
+                }
+                right = {"prefix": "", "contents": title_r, "suffix": ""}
+                self.print_layout(self.indent_detail, left, right)
+            else:
+                print_(self.indent_detail + "*", "Title:", title_r)
+
+    def make_medium_info_line(self, track_info):
+        """Construct a line with the current medium's info."""
+        media = self.match.info.media or "Media"
+        # Build output string.
+        if self.match.info.mediums > 1 and track_info.disctitle:
+            return f"* {media} {track_info.medium}: {track_info.disctitle}"
```

Comment:
Possible bug in this function when multiple mediums are in one release. See #3721